srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 5 nodes, setting nnodes to 1
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433434.9
2160
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433434.5
srun: Terminating job step 9433434.8
srun: error: sh02-08n48: task 0: Exited with exit code 1
srun: Terminating job step 9433434.28
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433434.6
srun: error: sh02-08n48: task 0: Exited with exit code 1
srun: Terminating job step 9433434.29
srun: error: sh02-08n48: task 0: Exited with exit code 1
srun: error: sh02-08n48: task 0: Exited with exit code 1
srun: Terminating job step 9433434.27
srun: error: sh02-01n10: task 0: Exited with exit code 1
srun: Terminating job step 9433434.26
srun: Terminating job step 9433434.7
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433434.3
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433434.4
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433434.1
srun: Terminating job step 9433434.0
srun: error: sh02-01n09: task 0: Exited with exit code 1
srun: Terminating job step 9433434.2
2160
2160
2160
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n23: task 0: Exited with exit code 1
srun: error: sh02-01n23: task 0: Exited with exit code 1
srun: Terminating job step 9433434.24
srun: Terminating job step 9433434.25
srun: error: sh02-01n23: task 0: Exited with exit code 1
srun: Terminating job step 9433434.23
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
2160
2160
2160
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
2160
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: Terminating job step 9433434.22
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: Terminating job step 9433434.17
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: Terminating job step 9433434.20
srun: Terminating job step 9433434.10
srun: Terminating job step 9433434.21
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: Terminating job step 9433434.14
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: Terminating job step 9433434.13
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: Terminating job step 9433434.19
srun: Terminating job step 9433434.11
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: Terminating job step 9433434.18
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: error: sh02-01n11: task 0: Exited with exit code 1
srun: Terminating job step 9433434.16
srun: Terminating job step 9433434.12
srun: Terminating job step 9433434.15
